lstchain/calib/camera/pedestals.py

```diff
@@ -71,6 +80,21 @@ def __init__(self, subarray, **kwargs):
         self.charges = None  # charge per event in sample
         self.sample_masked_pixels = None  # pixels tp be masked per event in sample
 
+        # declare the charge sampling corrector
+        if self.time_sampling_correction_path is not None:
+            # search the file in resources if not found
+            if not os.path.exists(self.time_sampling_correction_path):
```

Comment:
I commented on this before and you said you fixed it, but this still silently ignores a user configuration.

Reply:
Hi @maxnoe,
this PR is outdated, the code is (with the corrections I promised) in the newsreader_compatible branch. I am sorry, this is surely not the right way of proceeding but things went fast and somehow overlooked that the time sampling code was still not merged. I would suggest to close this PR.